internal/output/output_result.go

```diff
@@ -831,3 +832,19 @@ func getInstalledVersionOrCommit(pkg PackageResult) string {
 
 	return result
 }
+
+func isOSResult(sourceName string) bool {
+	return strings.Split(sourceName, ":")[0] == OSSourcePrefix
```

Comment:
I'm not a fan of doing a string split here. Can we pass through the enum and match on that directly?

Reply:
updated